packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -638,6 +685,30 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         }
     }
 
+    /**
+     * Returns the GC details in initial summary for the channel with the given id. The initial summary of the data
+     * store contains the GC details of all the child channel contexts that were created before the summary was taken.
+     * We find the GC details belonging to the given channel context and return it.
+     * @param channelId - The id of the channel context that is asked for the initial GC details.
+     * @returns the requested channel's GC details in the initial summary.
+     */
+    private async getChannelInitialGCDetails(channelId: string): Promise<IGarbageCollectionSummaryDetails> {
```

Comment:
I assume we have reasonable test coverage for new logic through existing UTs, right?

Reply:
Unfortunately, we do not have unit tests for data store runtime! and for channel contexts too. We should definitely add those.

Reply:
Created #5057 to add testing for these layers.